src/DetectChanges/BCBreak/ClassBased/MethodChanged.php

```diff
@@ -38,7 +35,7 @@ public function __invoke(ReflectionClass $fromClass, ReflectionClass $toClass):
      */
     private function checkSymbols(array $from, array $to): iterable
     {
-        foreach (array_keys(array_intersect_key($from, $to)) as $name) {
+        foreach (Vec\keys(Dict\intersect_by_key($from, $to)) as $name) {
```

Comment:
Similar to other comments about `array_keys()` and `array_intersect_key()`, I would suggest keeping this as it was